networking_ccloud/ml2/agent/eos/switch.py

```diff
@@ -105,11 +111,30 @@ class EOSSwitch(SwitchBase):
     # PL-CC-CLOUD02 | PL-CC-CLOUD02-A | PL-CC-CLOUD02-EXTERNAL | PL-CC-CLOUD02-A-EXTERNAL
     PREFIX_LIST_RE = re.compile("PL-(?P<vrf>.*?)(?:-(?P<az>[A-Z]))?(?:-(?P<external>EXTERNAL))?$")
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self._reset_version_detection()
+
+    def _reset_version_detection(self):
+        self._version_min_4_32 = None
+
+    @property
+    def version_min_4_35(self):
```

Comment:
why is the function call version_min_4_35? it's not obvious...

Reply:
We're in the switch class and the idea was that with this property we have a flag to check if the switch version we can expect is at minimum 4.35. I roughly copied this from what I did in the asr1k driver, but I'd be open for other naming ideas. Do you have any at hand that you think could be more obvious?

Reply:
would `version_gt_4_32` break anything? It's a bit odd to use the version in 4.35 in the header but then the implementation inside refers to 4_32

Reply:
Ooooh, I completely overlooked the 4.35 vs 4.32 part. I just mistyped the version in the header and reused the name. I'll fix it.